src/layout/_page.scss

```diff
@@ -14,3 +14,7 @@
     padding: ms(10) ms(2);
   }
 }
+
+.page-section--compact {
```

Comment:
Should we be chaining modifier onto their module? i.e. ```.page-section.page-section--compact { ... }``` to increase specificity.